unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +34,23 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist) {
+    if (first_playlist.size() != second_playlist.size()) {
+        return false;
+    }
+    std::unordered_multiset<std::string> playlist1 {}, playlist2 {};
+    for(auto const& song: first_playlist) {
+        playlist1.insert(song.name());
+    }
+    for(auto const& song: second_playlist) {
+        playlist2.insert(song.name());
+    }
+    for(auto const& song: playlist1) {
+        auto it = std::find(playlist2.begin(), playlist2.end(), song);
```

Comment:
please avoid this linear search

Reply:
I tried to think of another to do it like to use count to see if all the song from first playlist are contained in the second one. But also here using count method I will need to iterate through one of the playlists. Could you please provide me with an advice in order to avoid the use of this linear search? Thanks!

Reply:
See https://en.cppreference.com/w/cpp/container/unordered_multiset/operator_cmp
It will do it automatically, using the equal_range function